typescript/sdk/src/token/read.ts

```diff
@@ -0,0 +1,138 @@
+import { ethers, providers } from 'ethers';
+
+import {
+  ERC20__factory,
+  HypERC20Collateral__factory,
+} from '@hyperlane-xyz/core';
+import { ERC20Metadata, ERC20RouterConfig } from '@hyperlane-xyz/sdk';
+import { Address } from '@hyperlane-xyz/utils';
+
+import { DEFAULT_CONTRACT_READ_CONCURRENCY } from '../consts/crud.js';
+import { EvmHookReader } from '../hook/read.js';
+import { EvmIsmReader } from '../ism/read.js';
+import { MultiProvider } from '../providers/MultiProvider.js';
+import { ChainName } from '../types.js';
+
+type WarpRouteBaseMetadata = Record<
+  | 'mailbox'
+  | 'owner'
+  | 'wrappedTokenAddress'
```

Comment:
where is `wrappedTokenAddress` coming from? can't see it in the config types

Reply:
It should be `token` instead. Updated